src/core/Elsa.Abstractions/Services/Workflows/NullActivityPropertyValueProvider.cs

```diff
@@ -9,5 +9,9 @@ public class NullActivityPropertyValueProvider : IActivityPropertyValueProvider
         public static readonly NullActivityPropertyValueProvider Instance = new();
         public string RawValue => "null";
         public ValueTask<object?> GetValueAsync(ActivityExecutionContext context, CancellationToken cancellationToken = default) => new(null);
+        public ValueTask<bool> IsNonStorablePropertyValue(ActivityExecutionContext context, CancellationToken cancellationToken = default)
+        {
+            return new ValueTask<bool>(false);
+        }
```

Comment:
The primary motivation behind introducing the new domain event was to circumvent modifications to core contracts like these. If achieving this without altering the core contracts remains unfeasible, I'd appreciate a detailed explanation as to why that's the case.

Reply:
Sorry, it seems I forgot to delete this.